Doc/tutorial/modules.rst

```diff
@@ -339,7 +339,7 @@ Without arguments, :func:`dir` lists the names you have defined currently::
    >>> import fibo
    >>> fib = fibo.fib
    >>> dir()
-   ['__builtins__', '__name__', 'a', 'fib', 'fibo', 'sys']
+   ['__annotations__', '__builtins__', '__doc__', '__loader__', '__name__', '__package__', '__spec__', 'a', 'fib', 'fibo', 'sys']
```

Comment:
Is the example suggesting this is a fresh REPL session? If so, `sys` is not imported.

```suggestion
   ['__annotations__', '__builtins__', '__doc__', '__loader__', '__name__', '__package__', '__spec__', 'a', 'fib', 'fibo']
```